src/ComboControls.ts

```diff
@@ -259,7 +258,8 @@ export default class ComboControls extends EventDispatcher {
     y = (y / domElement.clientHeight) * -2 + 1;
 
     const dollyIn = delta < 0;
-    const deltaDistance = this.camera instanceof PerspectiveCamera ?
+    const deltaDistance =
+      this.camera.constructor.name === 'PerspectiveCamera' ?
```

Comment:
Why didn't `instanceof` work? We can't use `constructor.name` in production since this can change when the javascript file gets minified.